### PR TITLE
Setting default theme to bear_coat in install file.

### PR DESCRIPTION
**Pull request by sk33lz**

### CODE REVIEW

File: .probo.yaml [hunk 1]
```diff
@@ -4,5 +4,7 @@ steps:
     makeFile: build-bear.make
     profileName: bear
     runInstall: true
+  - name: Install bear_skin dependencies
```
Comment (jodyHamilton) [inline .probo.yaml]: aren't these bear_coat dependencies?

File: .probo.yaml [hunk 2]
```diff
@@ -4,5 +4,7 @@ steps:
...
+    command: 'cd /var/www/html/sites/all/libraries ; wget https://github.com/Semantic-Org/Semantic-UI/archive/2.1.6.zip; unzip 2.1.6.zip ; mv 2.1.6 semantic_ui ; cp -Rf semantic_ui/dist/* semantic_ui/.'
   - name: Run behat tests
```
Comment (jodyHamilton) [inline .probo.yaml:7]: Include this in build-a-bear.make not in probo

File: LICENSE.txt [hunk 3]
```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (jodyHamilton) [inline LICENSE.txt]: This stuff looks like it belongs in a separate pr

File: .probo.yaml (hunk 2 above)
Comment (sk33lz) [inline .probo.yaml:7]: It's not on the packaging whitelist :( I was trying to add it to get the dependencies working on a probo environment as I was getting that error about Semantic UI library.

File: LICENSE.txt (hunk 3 above)
Comment (sk33lz) [inline LICENSE.txt]: Ok I will move it to another.

File: .probo.yaml (hunk 1 above)
Comment (sk33lz) [inline .probo.yaml]: I wasn't sure at this point actually and said bear_skin thinking it was required for both bear_coat and bear_skin. After checking with Alban it's actually not for bear_skin so this is wrong. The packaging system blocking our libraries is blocking the building of the bear release properly on D.O. We may need to just live without bear_coat as the default and leave it bear_skin to avoid this issue. This code wouldn't be here if the semantic UI was on the packaging whitelist...

**Comment by sk33lz** [comment]
I've removed the LICENSE.txt file and the .probo.yaml step I was trying to use to get past errors on Probo using bear_coat. This should be clean, but I'm not sure we want to set this to default given the issues we are having currently.

**Comment by sk33lz** [comment]
It looks like Alban has removed semantic_ui requirement for bear_coat to be installed. It's now optional. I'm going to rebase this PR.
